CrossMod/FilteringOption.cs

```diff
@@ -13,13 +13,10 @@
 using Terraria.ModLoader;
 
 namespace MagicStorage.CrossMod {
-	public abstract partial class FilteringOption : ModTexturedType, ILocalizedModType
```

Comment:
`ILocalizedModType` usage should not have been removed.  please use the existing code in MagicStorage/1.4.4 as a comparison